src/components/collapsible-section/collapsible-section.scss

```diff
@@ -34,21 +39,33 @@ section {
         }
 
         &:hover {
-            background-color: #f8f8f8;
+            background-color: var(--limel-collapsible-section-header-open-background-color, #f8f8f8);
 
             > .expander {
-                color: #636363;
+                color: rgba(0,0,0,0.6);
             }
         }
     }
 
     > .body {
+        background-color: var(--limel-collapsible-section-body-background-color, #ffffff);
         display: none;
+        padding: var(--limel-collapsible-section-body-padding, 0);
+
+        &:before,
+        &:after {
+            content: " ";
+            display: table;
+        }
+
+        &:after {
+            clear: both;
+        }
```

Comment:
Clearfix baby!

Reply:
To clarify, after removing the section background colour on line 16, clearfixing was needed to ensure that the `.body` element contains any margins on descendant elements, or this would happen:

![image](https://user-images.githubusercontent.com/154725/73429915-0d188c00-433d-11ea-9b74-2cb7ca482be6.png)

Reply:
I'm not that familiar with clearfix, but I thought it was used to clear floats :thinking: I don't think we are using that here, right? Is it used for something else as well?

Reply:
Yeah, it clears the margins of the `<p>` element. Those are what jut out and creates the space between the body and the header in the screenshot above. They also jut out at the bottom of the body, but that's less apparent when looking at this screenshot. Compare to the screenshot on line 16 though, and you'll see that the bottom margin has been cleared there (and is thus contained within the body, instead of hanging outside it).